tests/acceptance/cli/ursula/test_ursula_command.py

```diff
@@ -74,10 +74,28 @@ def write(data: bytes):
     with capture_output() as (out, err):
         protocol.lineReceived(line=b'bananas')
 
+    commands = protocol.commands
+    hidden_commands = ['?', 'commit_next']
```

Comment:
Shouldn't we rather check for `commit-next` (with a hyphen)?

Reply:
there were underscores in interactive console for commands, so I just did the same (in CLI still hyphen)

Reply:
Ah, I see, I thought it was CLI.